### PR TITLE
[all] Hoist mocha

### PR DESCRIPTION
**Pull request by justinfagnani**

### CODE REVIEW

File: packages/web-component-tester/wct-browser-legacy/package.json [hunk 1]
```diff
@@ -33,7 +33,6 @@
     "async": "^1.5.2",
     "chai": "^3.5.0",
     "lodash": "^3.10.1",
-    "mocha": "^3.4.2",
```
[the remark this answers is not in the document]
Comment (justinfagnani) [reply packages/web-component-tester/wct-browser-legacy/package.json]: fixed

File: packages/web-component-tester/wct-browser-legacy/package-lock.json [hunk 2]
```diff
@@ -57,25 +57,6 @@
 			"resolved": "https://registry.npmjs.org/async/-/async-1.5.2.tgz",
```
Comment (rictic) [inline packages/web-component-tester/wct-browser-legacy/package-lock.json]: Update the wct-browser-legacy package-lock? Or, uh, un-update it?

Comment (justinfagnani) [reply packages/web-component-tester/wct-browser-legacy/package-lock.json]: yeah...

**Comment by justinfagnani** [comment]
@rictic PTAL, and let's hope tests go green

**Comment by rictic** [comment]
There's failures in bundler. These look like some changes I added that I wasn't certain about, where we needed to be more careful about the types of documents we had in a number of places.

It's possible that these tests were always failing, and the failures were just being swallowed previously.

**Comment by usergenic** [comment]
@rictic the failures are fixed in master. Vscode-uri had changed something that broke the way bundler was extending/generating new urla

**Comment by justinfagnani** [comment]
@rictic @usergenic just rebased on master to pick up those fixes

**Pull request merged**